code_gen/code_gen/generate_component_code.py

```diff
@@ -0,0 +1,43 @@
+import helpers.langchain_helpers as langchain_helpers
+import helpers.supabase_helpers as supabase_helpers
+from dotenv import load_dotenv
+load_dotenv()
+
+
+def main(app, prompt, verbose=False):
+    use_docs = True
+    db = supabase_helpers.SupabaseConnector()
+    row = db.client.table('components').select('*').match({'app_name_slug': app}).execute().data[0]
```

Comment:
Might be nice to break out all the `db.client.table` calls into different functions to improve the readability, e.g. 

```python
def get_app_docs(app):
    return db.client.table('components').select('*').match({'app_name_slug': app}).execute().data[0]
```